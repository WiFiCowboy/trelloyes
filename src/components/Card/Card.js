import React from 'react'

export default function Card(props) {
  console.log('card:',props)
  return (
    <div className="Card">
      <h3>{props.title}</h3>
      <p>{props.content}</p>
    </div>
  )
}
