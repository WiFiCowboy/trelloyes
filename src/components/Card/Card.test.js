import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';

describe('<Card />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('matches snapshot', () => {
    const divSnap = renderer.create(<Card />).toJSON();
    expect(divSnap).toMatchSnapshot();
  });
});

