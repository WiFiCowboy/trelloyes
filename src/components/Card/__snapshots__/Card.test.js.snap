// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> matches snapshot 1`] = `
<div
  className="Card"
>
  <h3 />
  <p />
</div>
`;
